pkg/apis/release/v1alpha1/releasepayload.go

```diff
@@ -0,0 +1,79 @@
+package v1alpha1
+
+import (
+	"sort"
+)
+
+// byCIConfigurationName sorts a list of CIConfigurations by their CIConfigurationName
+type byCIConfigurationName []CIConfiguration
+
+func (in byCIConfigurationName) Less(i, j int) bool {
+	return in[i].CIConfigurationName < in[j].CIConfigurationName
+}
+
+func (in byCIConfigurationName) Len() int {
+	return len(in)
+}
+
+func (in byCIConfigurationName) Swap(i, j int) {
+	in[i], in[j] = in[j], in[i]
+}
+
+// byJobStatusCIConfigurationName sorts a list of JobStatus' by their CIConfigurationName
+type byJobStatusCIConfigurationName []JobStatus
+
+func (in byJobStatusCIConfigurationName) Less(i, j int) bool {
+	return in[i].CIConfigurationName < in[j].CIConfigurationName
+}
+
+func (in byJobStatusCIConfigurationName) Len() int {
+	return len(in)
+}
+
+func (in byJobStatusCIConfigurationName) Swap(i, j int) {
+	in[i], in[j] = in[j], in[i]
+}
+
+// byCoordinatesName sorts a list of JobRunResults by their Coordinates Name
+type byCoordinatesName []JobRunResult
+
+func (in byCoordinatesName) Less(i, j int) bool {
+	return in[i].Coordinates.Name < in[j].Coordinates.Name
+}
+
+func (in byCoordinatesName) Len() int {
+	return len(in)
+}
+
+func (in byCoordinatesName) Swap(i, j int) {
+	in[i], in[j] = in[j], in[i]
+}
+
+// byLastTransitionTime sorts a list of ReleasePayloadConditions, descending, by their LastTransitionTime
+type byLastTransitionTime []ReleasePayloadCondition
+
+func (in byLastTransitionTime) Less(i, j int) bool {
+	return in[j].LastTransitionTime.Before(&in[i].LastTransitionTime)
+}
+
+func (in byLastTransitionTime) Len() int {
+	return len(in)
+}
+
+func (in byLastTransitionTime) Swap(i, j int) {
+	in[i], in[j] = in[j], in[i]
+}
+
+func (in *ReleasePayload) Canonicalize() {
```

Comment:
This is really `CanonicalizeReleasePayloadStatus`, right?

Reply:
Done